frontend/src/components/doctor/DetailDoctorPage.tsx

```diff
@@ -10,133 +10,112 @@ import {
+  }, []);
+
+  return individualDoctor ? (
     <Center py={6}>
       <Box
-        maxW={"320px"}
-        w={"full"}
+        // maxW={"320px"}
+        // w={"full"}
         // bg={useColorModeValue("white", "gray.900")}
         boxShadow={"2xl"}
         rounded={"lg"}
         p={6}
       >
-        {/* <Avatar
-          size={"xl"}
-          src={
-            "https://images.unsplash.com/photo-1520810627419-35e362c5dc07?ixlib=rb-1.2.1&q=80&fm=jpg&crop=faces&fit=crop&h=200&w=200&ixid=eyJhcHBfaWQiOjE3Nzg0fQ"
-          }
-          alt={"Avatar Alt"}
-          mb={4}
-          pos={"relative"}
-          _after={{
-            content: '""',
-            w: 4,
-            h: 4,
-            bg: "green.300",
-            border: "2px solid white",
-            rounded: "full",
-            pos: "absolute",
-            bottom: 0,
-            right: 3,
-          }}
-        /> */}
         <Heading fontSize={"2xl"} fontFamily={"body"} textAlign={"center"}>
-          {state.first_name} {state.last_name}
+          {individualDoctor.firstName} {individualDoctor.lastName}
         </Heading>
         <Text fontWeight={600} color={"gray.500"} mb={4} textAlign={"center"}>
-          {state.email}
+          {individualDoctor.email}
         </Text>
         <Text
         // color={useColorModeValue("gray.700", "gray.400")}
         >
           <Text as="span" fontWeight="semibold">
-            About me:
+            About:
           </Text>
-          {state.about}
+          {individualDoctor.about}
         </Text>
+
         <Flex>
-          {" "}
-          <Text fontWeight="semibold">Insurance:</Text>
-          <Text>{state.insurance}</Text>
-        </Flex>
-        <Flex>
-          <Text fontWeight="semibold">Speciality:</Text>
-          <Text>{state.speciality}</Text>
+          <Text fontWeight="semibold">Specialty:</Text>
+          <Text>{individualDoctor.specialty}</Text>
         </Flex>
 
-        <Text>
-          {" "}
-          <Text as="span" fontWeight="semibold">
-            Available Days:
-          </Text>
-          {state.days_available}
-        </Text>
         <Text>
           {" "}
           <Text as="span" fontWeight="semibold">
             Gender:
           </Text>
-          {state.gender}
+          {individualDoctor.gender}
         </Text>
 
         <Text>
           {" "}
           <Text as="span" fontWeight="semibold">
             Location:
           </Text>
-          {state.location}
+          {individualDoctor.location}
         </Text>
 
-        <Stack mt={8} direction={"row"} spacing={4}>
-          <Link
-            to={{
-              pathname: `/edit-doctor-details/${state.id}/${state.last_name}`,
+        {/* <Stack mt={8} direction={"row"} spacing={4}> */}
+        <Link
+          to={{
+            pathname: `/edit-doctor-details/${individualDoctor.userId}/${individualDoctor.lastName}`,
+          }}
+        >
+          <Button
+            display={"flex"}
```

Comment:
Small nitpick: You don't need curly braces for these prop values.
```suggestion
            display="flex"
```